docs/reference/mapping/types/text.asciidoc

```diff
@@ -96,6 +96,12 @@ The following parameters are accepted by `text` fields:
     the expense of a larger index. Accepts an
     <<index-prefix-config,`index-prefix configuration block`>>
 
+<<index-phrases,`index_phrases`>>::
+
+    If enabled, two-term word combinations ('shingles') are indexed into a separate
+    field.  This allows phrase queries to run more efficiently, at the expense
+    of a larger index.  Accepts `true` or `false` (default).
```

Comment:
Maybe we should document that this feature works better when stop words are not removed?

Reply:
+1